cmd/fuzzcrypto/main.go

```diff
@@ -95,6 +117,32 @@ func flagRegex(name, usage string) **regexp.Regexp {
 	return &reg
 }
 
+func flagBucket(name, usage string) (bucket, bucketCount *int) {
+	var b, c int
+	flag.Func(name, usage, func(s string) error {
+		before, after, found := strings.Cut(s, "/")
+		if !found {
+			return fmt.Errorf("no '/' found in %q", s)
+		}
+
+		var err error
+		b, err = strconv.Atoi(before)
+		if err != nil {
+			return fmt.Errorf("string before '/' is not a number: %v", err)
+		}
+		c, err = strconv.Atoi(after)
+		if err != nil {
+			return fmt.Errorf("string after '/' is not a number: %v", err)
+		}
+
+		if b < 1 || b > c {
+			return fmt.Errorf("bucket %v is not in range 1 - %v", b, c)
```

Comment:
Mathematical notation might be clearer here:

```suggestion
			return fmt.Errorf("bucket %v is not in range [1, %v]", b, c)
```

Reply:
Good idea. Can also call it an interval to point out the notation usage. 😄